package.json

```diff
@@ -72,6 +72,7 @@
     "babel-loader": "^8.0.5",
     "core-js": "^2.6.3",
     "tapable": "^1.1.1",
+    "terser": "^4.6.3",
```

Comment:
Please move this to `peerDependencies` (and add an entry in `devDependencies`) - lots of folks use `terser-webpack-plugin` which has its own dependency on `terser`, and weird things tend to happen when there are multiple versions of some packages in use in a given process. I'm also not _super_ nuts about the adding a direct dependency that's only used when a particular option is enabled.

Reply:
Okay. Tested the min working version to be 3.12.0 (typescript build fails before then).

There are no changes to the offline mirror since a version of terser already in there.